apps/wing/src/wingc.ts

```diff
@@ -25,12 +25,14 @@ export interface WingCompilerLoadOptions {
 
 export async function load(options: WingCompilerLoadOptions) {
   const preopens = {
+    "/": "/",
```

Comment:
idk if this is controversial, but we are not using WASM for it's security sandboxing right now. Anything limiting of filesystem access has to be handled in the runtime invoking wasm.

Reply:
Why is this needed?

Reply:
preopens grant the WASI runtime access to the host filesystem, and without some fancy analysis we don't know if the user will attempt to import libraries from anywhere on the system. 
This is also very useful for the lsp, which can open a file anywhere on the user's system at a moment's notice.

Reply:
Yeah I wasn't sure whether to add this to my bring jsii PR or not (security concerns and all) but I think it makes sense in the end. We could always add some kind of "secure" compile API/option later if needed